doc/reference/networking/thread.rst

```diff
@@ -0,0 +1,39 @@
+.. _thread_protocol_interface:
+
+Thread protocol
+###############
+
+Overview
+********
+Thread is a low-power mesh networking technology, designed specifically for home
+automation applications. It is an IPv6-based standard, which uses 6LoWPAN
+technology over IEEE 802.15.4 protocol. IP connectivity allows to easily connect
+a Thread mesh network to the internet with a Thread Border Router.
+
+Thread specification provides a high level of network security. Mesh networks
+built with Thread are secure - only authenticated devices can join the network
+and all communications within the mesh are encrypted. More information about
+Thread protocol can be found at
+`Thread Group website <https://www.threadgroup.org>`_.
+
+Zephyr integrates an open source Thread protocol implementation called OpenThread.
+See `OpenThread website <https://openthread.io/>`_ for more information about
+the OpenThread itself.
+
+Internet connectivity
+=====================
```

Comment:
I just wonder should the chapter use `*******` here, at least the other docs the Overview + other chapters are in the same level.

Reply:
Right, let's be consistent here.